scale/product/configuration/product_data_file.py

```diff
@@ -74,11 +76,17 @@ def _calculate_remote_path(self, job_exe, input_file_ids):
         :type input_file_ids: set of int
         :returns: The remote path for storing the products
         :rtype: str
-        '''
-
+        """
+
+        remote_path = ''
+        recipe = Recipe.objects.get_recipe_for_job(job_exe.job_id)
+        if recipe:
```

Comment:
You mentioned recipe/job name collisions and it isn't clear to me how this is handled? It seems like you would want something like a /recipes prefix and /jobs prefix to avoid this block of code having a conflict, or are you just assuming it will never actually happen?